hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/monkies/PolicyBasedChaosMonkey.java

```diff
@@ -116,13 +119,16 @@ public static <T> T selectWeightedRandomItem(List<Pair<T, Integer>> items) {
 
   /** Selects and returns ceil(ratio * items.length) random items from the given array */
   public static <T> List<T> selectRandomItems(T[] items, float ratio) {
-    int selectedNumber = (int) Math.ceil(items.length * ratio);
-
-    List<T> originalItems = Arrays.asList(items);
+    final int selectedNumber = (int) Math.ceil(items.length * ratio);
+    final List<T> originalItems = Arrays.asList(items);
```

Comment:
Arrays.asList will create a ArrayList which is backed by the given array, so the later shuffle will change the order of the items array which is passed in as a parameter. If we think this is acceptable, we can just shuffle on the given array instead of wrapping it? If not, I think we should copy the array.